apps/domain/lib/domain/telemetry.ex

```diff
@@ -44,8 +43,8 @@ defmodule Domain.Telemetry do
     :ok
   end
 
-  def add_user do
-    PostHog.capture("add_user", common_fields())
+  def add_actor do
+    PostHog.capture("add_actor", common_fields())
```

Comment:
Hm could we wait on changing the telemetry event names? This will require updating in PostHog to fix the investor metrics.

Let's think more about telemetry as we near release.

Reply:
If you want to leave it in to reduce cognitive load, that's fine -- maybe just comment out this module then with a `# TODO`.

Reply:
There is no way how telemetry will work the same with a new data model, that is so different. We can try to map things but this will lead to a mess, I would rather start clean with them.

Reply:
k